### PR TITLE
Party creation

### PR DESCRIPTION
**Pull request by manojpanta**
Created Party users table and set up all the relationship needed for users and party. Will be working on invitations next.

### CODE REVIEW

**Comment by Mackenzie-Frey** [comment]
@manojpanta Is this test failing on your machine? 
image

File: config/routes.rb [hunk 1]
```diff
@@ -1,9 +1,13 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  get 'party/create'
```
Comment (teresa-m-knowles) [inline config/routes.rb:4]: @manojpanta I think these two might have automatically generated from the scaffolded Party. Do we need them?

File: spec/models/party_spec.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  describe 'validations' do
+    it { should belong_to(:user)}
+    it { should belong_to(:party)}
+  end
+
+  describe 'party can have many users through' do
+    it 'party_users table' do
+      user = create(:user, name: 'manoj')
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
+      party = create(:party)
```
Comment (teresa-m-knowles) [inline spec/models/party_spec.rb]: test is failing because it should be 

```
party = create(:party, admin: user)
```

**Review by teresa-m-knowles — APPROVED** [review]
Looks great!

**Review by teresa-m-knowles — APPROVED** [review]

**Review by peregrinebalas — APPROVED** [review]

**Pull request merged**